build.gradle

```diff
@@ -47,8 +47,10 @@ project(':robotcode') {
    }
    repositories {
       jcenter()
+      mavenLocal()
```

Comment:
Pro Tip: mavenLocal() leads to inconsistent behavior between developers since makes assumptions about your local environment.  For our purposes, use mavenCentral() only.  Also comment why we're adding these repos.